modules/ROOT/pages/workday/workday-connector-design-center.adoc

```diff
@@ -1,63 +1,67 @@
-= Workday Design Center Configuration - Mule 4
+= Workday - Design Center Configuration - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
-
 :page-aliases: workday-design-center.adoc
 
-The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a Design environment. For more information, see the Flow Designer Tour.
+Anypoint Design Center's Flow Designer enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
 
 To create an app in Flow Designer:
 
 * Configure the input source (trigger) for your app.
-* Add the connector as a component to process the input for the app.
+* Configure the target component for your app.
 
 For information on Design Center field values, see
 the xref:workday/workday-reference.adoc[Workday Connector Reference].
 
-
 == Configure the Input Source Trigger
 
+A trigger starts your app. Workday Connector provides the *New Object Trigger* operation that starts your app when a new object is created in Workday. Alternatively, you can use the HTTP Listener operation to start your app from a browser 
+or with a command such as `curl`. Another alternative is to use Scheduler to start your app on a timed schedule.
+
 . In Design Center, click *Create*.
 . Click *Create new application*.
 . Specify a value for *Project name*, and click *Create*.
 . Click *Go straight to canvas* to exit from *Let's get started* and go to Flow Designer.
 . Click the trigger card.
-. If you are using your connector as an input source trigger, search for the connector;
+. If you are using the Workday connector as the input source trigger, search for the connector;
 otherwise, you can use HTTP or Scheduler.
-. Complete these fields when using *New objects trigger* as an input source trigger:
-
+. Complete these fields when using *New Object Trigger*:
 +
-[%header,cols="30s,55a,16a"]
+[%header,cols="30s,55a,15a"]
 |===
 |Field |Description |Required?
-|Configuration |The name of the configuration to use. |Yes
+|Configuration |Name of the configuration to use. |Yes
 |Object Types |Object type that triggers the poll. |Yes
-|Scheduling Strategy |Configures the scheduler that triggers the polling.|Yes
+|Scheduling Strategy |Scheduler configuration that triggers your app by polling.|Yes
 |Frequency |Polling frequency in time units.|No
-|Start delay|The delay time in time units that scheduler needs to wait before start.|No
-|Time Unit|The time unit for Frequency and Start delay values.|Yes
+|Start delay|Delay time in time units that Scheduler needs to wait before starting.|No
```

Comment:
We don't need "in time units" here. It is confusing, and the following entry clarifies that the time can be in different units, as specified:
```suggestion
|Start delay|Time that Scheduler waits before starting.|No
```